src/spdx/writer/tagvalue/package_writer.py

```diff
@@ -39,9 +39,10 @@ def write_package(package: Package, text_output: TextIO):
     write_value("PackageHomePage", package.homepage, text_output)
     write_text_value("PackageSourceInfo", package.source_info, text_output)
 
-    write_license_expression("PackageLicenseConcluded", package.license_concluded, text_output)
-    write_license_expression("PackageLicenseInfoFromFiles", package.license_info_from_files, text_output)
-    write_license_expression("PackageLicenseDeclared", package.license_declared, text_output)
+    write_value("PackageLicenseConcluded", package.license_concluded, text_output)
+    for license_info in package.license_info_from_files:
+        write_value("PackageLicenseInfoFromFiles", license_info, text_output)
```

Comment:
This causes a TypeError if snippet.license_info_in_snppet is SpdxNone or SpdxNoAssertion.

Reply:
fixed